api/lib/s2sTokenGeneration.ts

```diff
@@ -4,7 +4,7 @@ import { getConfigValue } from '../configuration'
 import { MICROSERVICE, S2S_SECRET } from '../configuration/references'
 import { http } from './http'
 import * as log4jui from './log4jui'
-import {valueOrNull} from "./util";
+import {valueOrNull} from "./util"
```

Comment:
@SivaprakasamSivaprakasam - this ought to be `import { valueOrNull } from '../lib/util'` (for consistency of spacing and use of single-quote marks)